packages/netlify-cms-backend-azure/src/API.ts

```diff
@@ -0,0 +1,782 @@
+import { Base64 } from 'js-base64';
+import { first, flow, partial, result, trim } from 'lodash';
+import {
+  localForage,
+  APIError,
+  ApiRequest,
+  unsentRequest,
+  responseParser,
+  Entry,
+  AssetProxy,
+  PersistOptions,
+  readFile,
+  CMS_BRANCH_PREFIX,
+  generateContentKey,
+  parseContentKey,
+  labelToStatus,
+  isCMSLabel,
+  EditorialWorkflowError,
+  statusToLabel,
+} from 'netlify-cms-lib-util';
+
+export const API_NAME = 'Azure DevOps';
+
+export class AzureRepo {
+  org: string;
+  project: string;
+  name: string;
+
+  constructor(location?: string | null) {
+    if (!location || location == undefined) {
+      throw new Error(
+        "An Azure repository must be specified in the format 'organisation/project/repo'.",
+      );
+    }
+
+    const components = trim(location, '/').split('/', 3);
```

Comment:
Would be nice to validate that location is indeed in the format `organisation/project/repo`